api/proto/logentry.proto

```diff
@@ -0,0 +1,126 @@
+// Copyright 2025 The Sigstore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: As of now this is a copy of https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_rekor.proto
```

Comment:
Can we adopt the protobuf-specs type now rather than later, given this was one of the primary reasons we decided on a photo-defined API? 

What are the differences? We don't have integrated_time, anything else?

Reply:
that's really it, they need to be made optional (or we fill them with junk for now).